e2e-tests/steps/login-steps.js

```diff
@@ -4,10 +4,10 @@ export class LoginSteps {
     }
 
     static loginWithUser(username, password) {
-        cy.get('.login-form').should('be.visible');
-        cy.get('#wb-login-username').type(username);
-        cy.get('#wb-login-password').type(password);
-        cy.get('#wb-login-submitLogin').click();
+        cy.get('#wb-login').should('be.visible');
+        cy.get('input[formControlName="username"]').type(username);
```

Comment:
Why do you select them like this?

Is this a new convention I missed, not to have any selectors on the elements?

I have even see here `data-cy` like attributes I think.

Reply:
done